docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+
+It is of course possible to explicitly target service shards in availability zones and regions other than the ones local to a Pod by specifying the appropriate DNS names explicitly, and not relying on automatic DNS expansion.  For example, "nginx.mynamespace.myfederation.svc.europe-west1.example.com" will resolve to all of the currently healthy service shards in Europe, even if the Pod issuing the lookup is located in the U.S., and irrespective of whether or not there are healthy shards of the service in the U.S.  This is useful for remote monitoring and other similar applications.
+
+### From Other Clients Outside your Federated Clusters
+
+Much of the above discussion applies equally to external clients, except that the automatic DNS expansion described is no longer possible.  So external clients need to specify one of the fully qualified DNS names of the Federated Service, be that a zonal, regional or global name. For convenience reasons, it is often a good idea to manually configure additional static CNAME records in your service, for example:
+```
+eu.nginx.acme.com        CNAME nginx.mynamespace.myfederation.svc.europe-west1.example.com.
+us.nginx.acme.com        CNAME nginx.mynamespace.myfederation.svc.us-central1.example.com.
+nginx.acme.com             CNAME nginx.mynamespace.myfederation.svc.example.com.
+```
+That way your clients can always use the short form on the left, and always be automatcally routed to the closest healthy shard on their home continent.  All of the required failover is handled for you automatically by Kubernetes Cluster Federation.  Future releases will improve upon this even further.
+
+## Handling Failures of Backend Pods and Whole Clusters
+
+Standard Kubernetes service cluster-IP's already ensure that non-responsive individual Pod endpoints are automatically taken out of service with low latency (a few seconds). In addition, as aluded above, the Kubernetes Cluster Federation system automatically monitors the health of clusters and the endpoints behind all of the shards of your Federated Service, taking shards in and out of service as required (e.g. when all of the endpoints behind a service, or perhaps the entire cluster or availability zone go down, or conversely recover from an outage).  Due to the latency inherent in DNS caching (the cache timeout, or TTL for Federated Service DNS records is configured to 3 minutes, by default, but can be adjusted), it may take up to that long for all clients to completely fail over to an alternative cluster in in the case of catastrophic failure. However, given the number of discrete IP addresses which can be returned for each regional service endpoint (see e.g. us-central1 above, which has three alternatives) many clients will fail over automatically to one of the alternative IP's in less time than that given appropriate configuration.
```

Comment:
Also, it might be useful to enable automatic line wrapping for markdown in your editor. Long paragraphs are too difficult to review.